config/software/agent-deps.rb

```diff
@@ -0,0 +1,57 @@
+require './lib/ostools.rb'
+
+name 'agent-deps'
+# Linux-specific dependencies
+if linux?
+  dependency 'procps-ng'
+  dependency 'sysstat'
+end
+# Ship supervisor anywhere but on Windows
+if not windows?
+  dependency 'supervisor'
+  dependency 'zlib'
+else
+  # We use our own supervisor shipped as a py2exe-built executable on Windows...
+  # therefore we need py2exe. We also need psutil for our home-made supervisor.
```

Comment:
I tried w/o psutil, and the build failed because it couldn't find psutil.  So I put it back in.